src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -872,8 +873,36 @@ internal override void Complete()
 
                     this.commandRuntime.PipelineProcessor.ExecutionFailed = true;
 
-                    if (!ExperimentalFeature.IsEnabled(ExperimentalFeature.PSNativeCommandErrorActionPreferenceFeatureName)
-                        || !Command.Context.GetBooleanPreference(SpecialVariables.PSNativeCommandUseErrorActionPreferenceVarPath, defaultPref: false, out _))
+                    // Feature is not enabled, so return
+                    if (!ExperimentalFeature.IsEnabled(ExperimentalFeature.PSNativeCommandErrorActionPreferenceFeatureName))
+                    {
+                        return;
+                    }
+
+                    // We send telemetry information only if the feature is enabled.
+                    // This shouldn't be done once, because it's a run-time check we should send telemetry every time.
```

Comment:
Is there a concern that this will create large ingress of data if this is sent for every native command used?

Reply:
I discussed this with @StevenBucher98 before I created the PR. While there is an opportunity to have an unexpected increase with ingress it seems that it would be good to be able to characterize how much it is. Further, we can measure the changes in our preview builds and then decide that it should not be done in non-preview builds.